metabase/metabase
# Combine filter header buttons

Closes https://github.com/metabase/metabase/issues/46192

### Description

Combines filter header buttons (to open filter modal and then to toggle the filter bar) into a single button.

![Screen Shot 2024-08-19 at 4 29 29 PM](https://github.com/user-attachments/assets/8aa34a5c-dfd4-481a-986d-8ec474a4132f)
![filterButton states](https://github.com/user-attachments/assets/e126ea46-9bd6-4631-9fd8-3b2a35b73374)

### How to verify

Open any question, add some filters, see the little button addon that lets you toggle the filter bar.

### Checklist
